### ISSUE TITLE
relation "zksync_batch_l2_transactions\" does not exist

### ISSUE THREAD

**Issue by oxf71** [blockscout/blockscout#9718]
### Description

zkevm is used but a `zksync_batch_l2_transactions does not exist` error is reported

```
2024-03-25T10:16:45.521922124+08:00 stdout F {"time":"2024-03-25T10:16:45.520Z","severity":"error","message":"#PID<0.19900.14> running BlockScoutWeb.Endpoint (connection #PID<0.18791.14>, stream id 2) terminated 
Server: habitat-backend.bsquared.network:80 (http)
Request: GET /api/v2/transactions/zkevm-batch/241\n** (exit) an exception was raised:
 ** (Postgrex.Error) ERROR 42P01 (undefined_table) relation \"zksync_batch_l2_transactions\" does not exist
 query: SELECT z0.\"batch_number\", z0.\"hash\", z0.\"inserted_at\", z0.\"updated_at\" FROM \"zksync_batch_l2_transactions\" AS z0 WHERE (z0.\"batch_number\" = $1)\n        (ecto_sql 3.11.1) lib/ecto/adapters/sql.ex:1054: Ecto.Adapters.SQL.raise_sql_call_error/1\n        
 (ecto_sql 3.11.1) lib/ecto/adapters/sql.ex:952: Ecto.Adapters.SQL.execute/6\n        
 (ecto 3.11.2) lib/ecto/repo/queryable.ex:232: Ecto.Repo.Queryable.execute/4\n        
 (ecto 3.11.2) lib/ecto/repo/queryable.ex:19: Ecto.Repo.Queryable.all/3\n        
 (block_scout_web 6.3.0) lib/block_scout_web/controllers/api/v2/transaction_controller.ex:171: BlockScoutWeb.API.V2.TransactionController.polygon_zkevm_batch/2\n        
 (block_scout_web 6.3.0) lib/block_scout_web/controllers/api/v2/transaction_controller.ex:1: BlockScoutWeb.API.V2.TransactionController.action/2\n        (block_scout_web 6.3.0) lib/block_scout_web/controllers/api/v2/transaction_controller.ex:1: BlockScoutWeb.API.V2.TransactionController.phoenix_controller_pipeline/2\n        
 (phoenix 1.5.14) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2","metadata":{"error":{"initial_call":null,"reason":"** (Postgrex.Error) ERROR 42P01 (undefined_table) relation \"zksync_batch_l2_transactions\" does not exist\n\n    
 query: SELECT z0.\"batch_number\", z0.\"hash\", z0.\"inserted_at\", z0.\"updated_at\" FROM \"zksync_batch_l2_transactions\" AS z0 WHERE (z0.\"batch_number\" = $1)\n    
 (ecto_sql 3.11.1) lib/ecto/adapters/sql.ex:1054: Ecto.Adapters.SQL.raise_sql_call_error/1\n    
 (ecto_sql 3.11.1) lib/ecto/adapters/sql.ex:952: Ecto.Adapters.SQL.execute/6\n    
 (ecto 3.11.2) lib/ecto/repo/queryable.ex:232: Ecto.Repo.Queryable.execute/4\n    
 (ecto 3.11.2) lib/ecto/repo/queryable.ex:19: Ecto.Repo.Queryable.all/3\n    
 (block_scout_web 6.3.0) lib/block_scout_web/controllers/api/v2/transaction_controller.ex:171: BlockScoutWeb.API.V2.TransactionController.polygon_zkevm_batch/2\n    
 (block_scout_web 6.3.0) lib/block_scout_web/controllers/api/v2/transaction_controller.ex:1: BlockScoutWeb.API.V2.TransactionController.action/2\n    (block_scout_web 6.3.0) lib/block_scout_web/controllers/api/v2/transaction_controller.ex:1: BlockScoutWeb.API.V2.TransactionController.phoenix_controller_pipeline/2\n    
 (phoenix 1.5.14) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2\n"}}}
```

### Type of the installation

Docker-compose

### Type of the JSON RPC archive node

polygonZkevm

### Type of the chain

L2

### Link to the page

https://habitat-explorer.bsquared.network/batches/265?tab=txs

### Backend version

v6.3.0

### Frontend version

v1.27.0

### Elixir & Erlang/OTP versions

Elixir

**Comment by vbaranov** [comment]
@oxf71 how did you run docker-compose?

If you use Polygon zkEVM JSON RPC node, you should set:
```
CHAIN_TYPE=polygon_zkevm
```

And use `blockscout/blockscout-zkevm:latest` docker image tag.

**Comment by vbaranov** [comment]
@oxf71 this has been fixed in master branch. So, the fix should be available with release 6.4.0, which is currently in QA.
